app/services/force/soql/preference_service.rb

```diff
@@ -55,59 +55,78 @@ def buildAppPreferencesFilters(opts)
         end
 
         filters
-    end
+      end
 
-    def buildAppPreferencesSearch(search_terms_string)
-      # Given a comma-separated list of search terms:
-      # For each term, we search across name and application number for a match
-      # For multiple terms, we expect the record to match each of the terms individually.
-      search_terms = search_terms_string.split(',')
-      search_fields = [
-        "Application__r.Name",
-        "Application__r.Applicant__r.First_Name__c",
-        "Application__r.Applicant__r.Last_Name__c",
-      ]
-      search = []
-      for search_term in search_terms
-        # For each term we join the fields with OR to return a record if it matches
-        # across any of these fields.
-        search_term_clause = []
-        sanitized_term = ActiveRecord::Base.sanitize_sql_like search_term
-        for search_field in search_fields
-          search_term_clause.push("#{search_field} like '%#{sanitized_term}%'")
+      def buildAppPreferencesSearch(search_terms_string)
+        # Given a comma-separated list of search terms:
+        # For each term, we search across name and application number for a match
+        # For multiple terms, we expect the record to match each of the terms individually.
+        search_terms = search_terms_string.split(',')
+        search_fields = [
+          'Application__r.Name',
+          'Application__r.Applicant__r.First_Name__c',
+          'Application__r.Applicant__r.Last_Name__c',
+        ]
+        search = []
+        for search_term in search_terms
```

Comment:
issue: follow Rubocop guidelines and use block-style looping, e.g. `search_terms.each do ... end`.

Reply:
I'm only addressing rubocop things that I wrote - would rather not update old code if I don't have to